io.github.purplehorrorrus.Meridius.json

```diff
@@ -48,7 +48,7 @@
                 {
                     "type": "file",
                     "url": "https://raw.githubusercontent.com/PurpleHorrorRus/Meridius/master/flatpak/io.github.purplehorrorrus.Meridius.metainfo.xml",
-                    "sha256": "1bc0f77b39f0c89530a3765894e8923e72cf5b0f6f75734d9a6275f3b7dcb222"
+                    "sha256": "73e59ee002f9cd7f94c89db8b608cda7c85b77b02ffad25d05fa0f485dcbfac9"
```

Comment:
this will break as soon as the the file get changed in the repository.

Best: have it in the tarball.
Second best: reference a specific commit or tag.